Frameworks/UIKit.Xaml/Layer.h

```diff
@@ -0,0 +1,70 @@
+﻿//******************************************************************************
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
```

Comment:
nit: if we're splitting these out, we should probably call them ILayer.h/ILayer.cpp